packages/itwin/grouping-mapping-widget/src/components/Properties/CalculatedProperties/CalculatedPropertyAction.tsx

```diff
@@ -33,39 +33,45 @@ export const CalculatedPropertyAction = ({
   onClickCancel,
 }: CalculatedPropertyActionProps) => {
   const { getAccessToken, iModelId } = useGroupingMappingApiConfig();
-  const mappingClient = useMappingClient();
+  const propertiesClient = usePropertiesClient();
   const [propertyName, setPropertyName] = useState<string>(
     calculatedProperty?.propertyName ?? "",
   );
-  const [type, setType] = useState<CalculatedPropertyType | undefined>(calculatedProperty?.type);
+  const [type, setType] = useState<CalculatedPropertyType | undefined>(calculatedProperty?.calculatedPropertyType);
   const [validator, showValidationMessage] = useValidator();
   const queryClient = useQueryClient();
 
   const { mutate: saveMutation, isLoading } = useMutation(async (type: CalculatedPropertyType) => {
     const accessToken = await getAccessToken();
 
     return calculatedProperty
-      ? mappingClient.updateCalculatedProperty(
+      ? propertiesClient.updateProperty(
         accessToken,
-        iModelId,
         mappingId,
         group.id,
         calculatedProperty.id,
-        { propertyName, type },
+        {
+          propertyName,
```

Comment:
To not overwrite anything else (ecProperties, formula) on the property do `{ ...calculatedProperty, propertyName, calculatedPropertyType: type }`

Reply:
Done.

Reply:
I don't see the changes.

Reply:
Resolved.

Reply:
Resolved.